### DISCUSSION TITLE
Significant digits

### DISCUSSION THREAD

**Discussion by Fumale** [frangoteam/FUXA#419] [Q&A]
Hello everybody,
Is there a way to display only the significant digits of a tag?
For example :
Tag raw value -> 123.4567890
Display -> 123.46

**Comment by 2345lug** [comment]
I think its important question for all. How to use format of numbers?

**Comment by unocelli** [comment]
Hi, currently not very consistent, several solutions have been tried, I believe that the format setting as in the table is the most intuitive   
<img width="597" alt="image" src="https://user-images.githubusercontent.com/48684550/192111946-945de755-d039-4ae4-92ae-abb3ed6d0888.png">
